standardtracer/spanpropagator.go

```diff
@@ -135,7 +136,7 @@ func (s *tracerImpl) JoinTraceFromBinary(
 				SpanID:       randomID(),
 				ParentSpanID: propagatedSpanID,
 				Sampled:      sampledByte != 0,
-				traceAttrs:   make(map[string]string),
```

Comment:
yikes -- obviously this code was never tested (I was more trying to convince myself that this stuff could be written sanely).